pkg/config/log.go

```diff
@@ -350,10 +350,25 @@ func parseShortFilePath(params string) seelog.FormatterFunc {
 }
 
 func extractShortPathFromFullPath(fullPath string) string {
-	// We want to trim the part containing the path of the project
-	// ie DataDog/datadog-agent/ or DataDog/datadog-process-agent/
-	slices := strings.Split(fullPath, "-agent/")
-	return slices[len(slices)-1]
+	shortPath := ""
+	if strings.Contains(fullPath, "-agent/") {
+		// We want to trim the part containing the path of the project
+		// ie DataDog/datadog-agent/ or DataDog/datadog-process-agent/
+		slices := strings.Split(fullPath, "-agent/")
+		shortPath = slices[len(slices)-1]
+	} else {
+		// For logging from dependencies, we want to log e.g.
+		// "collector@v0.35.0/service/collector.go"
+		slices := strings.Split(fullPath, "/")
+		atSignIndex := len(slices) - 1
+		for ; atSignIndex > 0; atSignIndex-- {
+			if strings.Contains(slices[atSignIndex], "@") {
+				break
+			}
+		}
+		shortPath = strings.Join(slices[atSignIndex:], "/")
```

Comment:
This misses the package path, I think?  e.g., `/home/jenkins/workspace/process-agent-build-ddagent/go/src/github.com/foo/bar@1.2.3/bing/baz.go` -> `bar@1.2.3/bing/baz.go`